test/pop_protocol_test.erl

```diff
@@ -52,28 +54,42 @@ make_block(PrevId, Height, PrivateKey, PublicKey, Index, Time, Transactions) ->
 		    },
     Block1.
 
-
-
-basic_test() ->
+make_verifier_array() ->
     %% make verifier array, they share keys
     PrivateKey = my_crypto:read_file_key(private, "key1.prv"),
     PublicKey = my_crypto:read_file_key(public, "key1.pub"),
     VerFunc = fun(Index) -> #verifier_public_info{index = Index, public_key = PublicKey} end,
     VerifierArr = array:from_list(lists:map(VerFunc, [0, 1, 2, 3, 4])),
 
+    {VerifierArr, PrivateKey, PublicKey}.
```

Comment:
isn't this a tuple?

Reply:
yes?

Reply:
oh I see, because verifierArr is an element in the tuple.. I was about to suggest you rename it to make_verifier_tuple() D: